docs.kosli.com/content/tutorials/unauthorized_iac_changes.md

```diff
@@ -0,0 +1,166 @@
+bookCollapseSection: false
+weight: 506
+---
+
+# Detecting unauthorized Terraform IaC changes
+
+Authorized Terraform changes follow a predefined process that maintains a certain level of quality, security and safety for the underlying infrastructure. Unauthorized changes, however, can undermine the integrity and reliability of the infrastructure. Hence the importance of prompt detection of such changes.
+
+Unauthorized Terraform changes happen in one of two ways:
+1. Bypassing Terraform and making direct changes via cloud APIs, clients or UI consoles. This leads to drift, where the desired state does not match the actual state. This kind of unauthorized change can be detected and corrected with [Terraform drift detection](https://developer.hashicorp.com/terraform/tutorials/state/resource-drift).
+2. Bypassing the predefined process for Terraform changes. For example, a developer running terraform directly from their machine without going through CI.
+
+This tutorial shows how you can use Kosli to track and detect the second type of unauthorized changes.
+
+## Prerequisites
+
+To follow the steps in this tutorial, you need to:
+* [Install Terraform on your machine](https://developer.hashicorp.com/terraform/install).
+* (Optional)[Setup Snyk on your machine](https://docs.snyk.io/snyk-cli/getting-started-with-the-snyk-cli#install-the-snyk-cli-and-authenticate-your-machine).
+* [Create a Kosli account](https://app.kosli.com/) (Skip if you already have one).
+* [Install Kosli CLI](/getting_started/install/).
+* [Get a Kosli API token](/getting_started/service-accounts/).
+* Set the `KOSLI_ORG` environment variable to your personal org name and `KOSLI_API_TOKEN` to your token:
+  ```shell {.command}
+  $ export KOSLI_ORG=<your-personal-kosli-org-name>
+  $ export KOSLI_API_TOKEN=<your-api-token>
+  ```
+* Clone the tutorial git repo
+  ```shell {.command}
+  $ git clone https://github.com/kosli-dev/iac-changes-tutorial.git 
+  $ cd iac-changes-tutorial
+  ```
+
+## Creating a Kosli flow
+
+We will start by creating a Kosli flow to represent the process for authorized Terraform changes.
+For simplicity, we will not define any requirements for this process by using `--use-empty-template`
+
+```shell {.command}
+$ kosli create flow tf-tutorial --use-empty-template
+```
+
+## Making and tracking an authorized change
+
+{{<hint info>}}
+In production, an authorized change will normally go though CI.
+In this tutorial, however, we run the commands that you would otherwise do in CI locally for simplicity.
+{{</hint>}}
+
+Let's create a trail to represent a single instance of making an authorized change. We will call it `authorized-1`.
+
+```shell {.command}
+$ kosli begin trail authorized-1 --flow tf-tutorial
+```
+Next, we can scan our terraform config scripts for security issues. We capture the SARIF output from the scan and attest it to Kosli.
+
+```shell {.command}
+$ snyk iac test main.tf --sarif-file-output=sarif.json
```

Comment:
Very small thing - the snyk command uses the `--flag=value` syntax.
The Kosli commands dont use the `=` 
Should we be consistent?
Personally I find having the `=` a little bit more readable.

Reply:
done